Vostok.Logging.File/Rolling/Strategies/HybridRollingStrategy.cs

```diff
@@ -13,27 +13,30 @@ internal class HybridRollingStrategy : IRollingStrategy
         private readonly Func<DateTime> timeProvider;
         private readonly IFileSuffixFormatter<DateTime> timeSuffixFormatter;
         private readonly IFileSuffixFormatter<(DateTime, int)> hybridSuffixFormatter;
+        private readonly char suffixEliminator;
 
         public HybridRollingStrategy(
             IFileSystem fileSystem, 
             IRollingStrategy sizeRollingStrategy, 
             Func<DateTime> timeProvider, 
             IFileSuffixFormatter<DateTime> timeSuffixFormatter, 
-            IFileSuffixFormatter<(DateTime, int)> hybridSuffixFormatter)
+            IFileSuffixFormatter<(DateTime, int)> hybridSuffixFormatter,
+            char suffixEliminator = '-')
```

Comment:
Please remove the default value. There are only 2 places where this ctor is invoked and it's just not worth the risk to forget specifying it in the future. This comment also applies to other strategies (TimeBased, SizeBased), as well as HybridSuffixFormatter.